src/dpos/store/index.ts

```diff
@@ -382,3 +385,49 @@ async function claimRewards(context: ActionContext) {
     feedback.showError("Error while claiming rewards. Please contact support.")
   }
 }
+
+/**
+ * NOTE: make sure candiate.pubKey corresponds to rootState.plasma.address
+ * @param context
+ * @param candidate
+ */
+export async function registerCandidate(context: ActionContext, candidate: ICandidate) {
+  const balance = context.rootState.plasma.coins.LOOM.balance
+  const weiAmount = parseToWei("1240000")
```

Comment:
We should put this in config, or see if there is a method on the contract to get this value. But it's ok for now.